rules/CodeQuality/Rector/Assign/CombinedAssignRector.php

```diff
@@ -48,6 +55,14 @@ public function getNodeTypes(): array
      */
     public function refactor(Node $node): ?Node
     {
+        try {
+            $class = $this->reflectionProvider->getClass("ClassWhichNeedsShouldTriggerAutoloading");
```

Comment:
here we try to use the reflectionProvider on a class which does not exist.
my understand is that this should invoke the configured autoloader in https://github.com/rectorphp/rector/pull/6158/files#diff-ee5a9cb644ef1a427edfa91f0d8656e988995c358f98656f9d83e5aefccd8f6dR13